docs/source/download.rst

```diff
@@ -151,35 +151,30 @@ Windows Subsystem for Linux. There are more detailed
 `instructions here <https://github.com/firedrakeproject/firedrake/wiki/Installing-on-Windows-Subsystem-for-Linux>`_.
 Installation on previous versions of Windows is unlikely to work.
 
+
+Anaconda
+~~~~~~~~
+
+.. warning::
+    The following is recent as of Sept 2022 and still highly experimental.
+
+Firedrake was incompatible with the Anaconda Python distribution for a
```

Comment:
Would this work with https://conda-forge.org/, specifically their https://github.com/conda-forge/miniforge installer? (That's what I use to ship all my stuff.)

Conda-forge has MPI and PETSc precompiled, which may be convenient for shipping to laptops without forcing folks to wait through the PETSc compile.

https://github.com/conda-forge/petsc-feedstock

Reply:
This hits the usual problem with prebuilt PETSc, which is that the conda-forge PETSc isn't built with all the configure options that Firedrake depends on.

Reply:
Conda (and conda forge) support build variants, you could depend on a specific one.

Reply:
So there's a variant which has what we need? I think I'm not getting this.

Reply:
Maybe? The point is that conda-forge is a community distribution, and a build variant that meets your specific needs is potentially one pull request to https://github.com/conda-forge/petsc-feedstock away.

Reply:
Ah. Ok. Well once we get through this postdoc hiring round we officially have the funded resource to get better at being installed so this is a serious option.